setup.cfg

```diff
@@ -27,13 +27,19 @@ install_requires =
     tomli-w
     pydantic
     loguru
+    bmi-python
+    HYDROLIB-core
+    pandas
+    pywin32
+    xarray
```

Comment:
hydrolib, pandas and xarray are only a test requirement.

Reply:
done